x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/curation_queries/curation_queries_logic.test.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { resetContext } from 'kea';
+
+import { CurationQueriesLogic } from './curation_queries_logic';
+
+describe('CurationQueriesLogic', () => {
+  const MOCK_QUERIES = ['a', 'b', 'c'];
+
+  const DEFAULT_PROPS = { queries: MOCK_QUERIES };
+  const DEFAULT_VALUES = {
+    queries: MOCK_QUERIES,
+    hasEmptyQueries: false,
+    hasOnlyOneQuery: false,
+  };
+
+  const mount = (props = {}) => {
```

Comment:
Don't we have a mount helper we typically use?

Reply:
It doesn't include a way to pass props, and I very specifically wanted to ensure props was fully tested/used rather than setting state via defaults. None of the other shared logic files that instantiate via props use the mount helper either FWIW 🤷

Reply:
Thinking over this a bit more, we might be able to refactor the LogicMounter helper to instantiate with a flag that passes props rather than defaults - something like `new LogicMounter(SomeFile, { props: true })`? 🤔 Will let it sit for a bit but might punt on this for now